### PR TITLE
Add Derivative: Loot Dungeon

### PR DESCRIPTION
**Pull request by dmerrill6**
## The first game built on Loot

Through smart contracts you can stake your Loot and battle a monster. Battle results are determined by a dice roll (using Chainlink VRFs) and your Loot stats. By winning battles you can claim unique and rare items. Each monster drops a guaranteed item and might drop a rare card. Items are ERC1155 NFTs containing machine-generated pixel art.

Website: https://lootdungeon.app
Etherscan: https://etherscan.io/address/0xdc9d70e37662ce16615224efaa9bb2315b80e36b
OpenSea: https://opensea.io/collection/loot-dungeon
Twitter: https://twitter.com/LootDungeonApp
Discord: https://discord.gg/z6azBRgZ3J

### CODE REVIEW

**Comment by ravindranrahul** [comment]
Can you please resolve conflicts

**Comment by dmerrill6** [comment]
@ravindranrahul Done! Thanks.

File: utils/lists.ts [hunk 1]
```diff
@@ -352,6 +352,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Every adventurer needs a Dicc!",
     url: "https://etherscan.io/address/0xfecc1e1449496c0cddfb1a075e0ef74c50538c1a",
   },
+  {
+    name: "Loot Dungeon",
+    description: "D&D-inspired game that lets you fight monsters using your Loot to earn NFT rewards",
```
Comment (ravindranrahul) [inline utils/lists.ts:357]: Awesome. Can you please make the description shorter so it fits in one line like others? I think that should be it

**Review by ravindranrahul — CHANGES_REQUESTED** [review]

File: utils/lists.ts (hunk 1 above)
Comment (dmerrill6) [reply utils/lists.ts]: @ravindranrahul Fixed! Thanks!

**Review by ravindranrahul — APPROVED** [review]

**Comment by ravindranrahul** [comment]
Thanks for your contribution

**Pull request merged**